code/game/objects/items/storage/backpack.dm

```diff
@@ -766,3 +766,127 @@
 	desc = "A rucksack with origins dating back to the mining colonies."
 	icon_state = "som_lightpack"
 	item_state = "som_lightpack"
+
+/obj/item/storage/backpack/dispenser
+	name = "TX-9000 Provisions Dispenser"
+	desc = "The TX-9000 also known as \"Dispenser\" is a machine capable of holding a big amount of items on it, while also healing nearby humans. Your allies will often ask you to lay down one of those."
+	icon = 'icons/obj/items/storage/storage_48.dmi'
+	icon_state = "dispenser"
+	flags_equip_slot = ITEM_SLOT_BACK
+	max_w_class = 6
+	max_storage_space = 63
+	///list of human mobs we're currently affecting in our area.
+	var/list/mob/living/carbon/human/affecting_list
```

Comment:
```suggestion
	var/list/mob/living/carbon/human/affecting_list = list()
```

Reply:
no need to initialize the list since it already is done so later on, leaving it as null saves memory, also (ours and tg contrubution guideline on this) declaring a list on variables causes it to init the object twice

Reply:
well not init twice, it causes it to call a hidden proc called init() to make the lists and then it calls New() after so New() has access to the list filled